frontend/www/js/omegaup/components/problem/BaseList.vue

```diff
@@ -303,6 +303,11 @@ export default class BaseList extends Vue {
   top: 4rem;
 }
 
+table {
+  border-collapse: separate;
```

Comment:
```suggestion
  border-collapse: collapse;
```

qué pasa si usamos esto? con eso ya no sería necesario L320?

Reply:
Si se usa `border-collapse: collapse` es cuando las líneas entre filas desaparecen.